### PR TITLE
Compatibility with newer mediasource interface

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/NEX-1202

depends on https://github.com/oat-sa/extension-tao-itemqti/pull/1472/

### CODE REVIEW

File: model/MediaSource.php [hunk 1]
```diff
@@ -177,8 +180,8 @@ public function getFileInfo($link)
      */
     public function getFileStream($link)
     {
-        $resource = new \core_kernel_classes_Resource(\tao_helpers_Uri::decode($link));
-        $fileLink = $resource->getOnePropertyValue(new \core_kernel_classes_Property(MediaService::PROPERTY_LINK));
+        $resource = $this->getResource(tao_helpers_Uri::decode($link));
+        $fileLink = $resource->getOnePropertyValue($this->getProperty(MediaService::PROPERTY_LINK));
```
Comment (gabrielfs7) [inline model/MediaSource.php:184]: What happen if you add to the end of this method? Maybe you do not need the second method?

```
        return stream_for(
            $this->getPreparer()->prepare(
                $this->getResource(tao_helpers_Uri::decode($link)),
                $this->getFileManagement()->getFileStream($fileLink)
            )
        );
```

File: manifest.php [hunk 2]
```diff
@@ -28,10 +28,10 @@
     'label' => 'extension-tao-mediamanager',
     'description' => 'TAO media manager extension',
     'license' => 'GPL-2.0',
-    'version' => '10.0.2',
+    'version' => '11.0.0',
```
Comment (gabrielfs7) [inline manifest.php:31]: - [ ] Please, solve the conflict with manifest and updater too.

File: scripts/install/SetMediaManager.php [hunk 3]
```diff
@@ -69,8 +68,6 @@ public function __invoke($params)
         /** @var MediaService $mediaService */
         $mediaService = $this->getServiceManager()->get(MediaService::SERVICE_ID);
         $mediaService->addMediaSource(new MediaSource());
-        $mediaService->setOption(MediaService::OPTION_PREPARER, new MediaResourcePreparer());
-        $this->getServiceManager()->register(MediaService::SERVICE_ID, $mediaService);
```
Comment (bartlomiejmarszal) [inline scripts/install/SetMediaManager.php:73]: u don't need this?

Comment (boajer) [reply scripts/install/SetMediaManager.php:73]: Not needed anymore, as that was temporary solution for the pilot; cleaned up in `tao`, so need to update runtime configurations

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**